tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/binaryoperations/TypeTestExprTest.java

```diff
@@ -391,6 +391,11 @@ public void testObjectIsCheckWithCycles() {
         BRunUtil.invoke(result, "testObjectIsCheckWithCycles");
     }
 
+    @Test(description = "Test is-expr with service type objects")
+    public void testServiceObjects() {
+        BRunUtil.invoke(result, "testServiceObjects");
```

Comment:
We have several other tests that have the same `BRunUtil.invoke(result, <function-name>);` approach. Shall we move them to a data provider?
Like this, 
https://github.com/ballerina-platform/ballerina-lang/blob/ebae8819aed845fb7a3c53e3508d635f2fd2faa9/tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/binaryoperations/MultiplyOperationTest.java#L81

Reply:
I will look into this.